canonical-web-and-design/juju.is
# Update quotations on k8s report page

## Done
Updated the quotation style on the k8s survey report page

## QA
- Go to https://juju-is-278.demos.haus/cloud-native-kubernetes-usage-report-2021
- Check that the pull quotes match [the design](https://app.zeplin.io/project/606203889af57e12287b5308/screen/60ca0ee03cc6b0bc46afaa93)

## Issue
Fixes #274
